deployment/ansible/group_vars/all.example

```diff
@@ -34,5 +34,8 @@ redis_port: 6379
 web_js_nominatim_key: ""
 web_js_html5mode: "true"
 web_js_html5mode_prefix: "!"
+web_js_recordType_visible: "false"
```

Comment:
The mix of camel and snake case seems strange; might be good to rename for consistency